bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -47,6 +47,8 @@ spec:
       name: cert-dir
     - mountPath: /var/log/kube-apiserver
       name: audit-dir
+    - mountPath: /etc/pki/ca-trust/extracted/pem/
```

Comment:
is this overriding existing directory? in case there is no trusted-ca-bundle config map, we erase that dir?

Reply:
(i wonder if this should be handled in the installer pod controller)

Reply:
That's a good point, I think the hostPath directory will only appear once the appropriate CM exists. I guess setting `type: Directory` for the matching `volume` might solve the problem of the `hostPath` directory not existing.

Reply:
@stlaz the mount should be optional.

Reply:
@danehans I don't think there is a way how that could work here

Reply:
Nvm, we found a way around it to make it optional